app/MMB/src/scenes/passenger/Passenger.js

```diff
@@ -0,0 +1,121 @@
+// @flow
+
+import * as React from 'react';
+import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
+import { Translation, DateFormatter } from '@kiwicom/mobile-localization';
+import { SimpleCard, StyleSheet, Text, Color } from '@kiwicom/mobile-shared';
+import idx from 'idx';
+import { View } from 'react-native';
+import startCase from 'lodash/startCase';
+
+import { SeparatorFullWidth } from '../../components/Separators';
+import type { Passenger_passenger as PassengerType } from './__generated__/Passenger_passenger.graphql';
+
+type RowItemProps = {|
+  title: React.Element<typeof Translation>,
+  value: React.Element<typeof Translation>,
+|};
+
+const RowItem = (props: RowItemProps) => (
+  <View style={styles.rowItem}>
+    <Text style={styles.titleLight}>{props.title}</Text>
+    {props.value}
+  </View>
+);
+
+type Props = {|
+  passenger: PassengerType,
+|};
+
+export class Passenger extends React.Component<Props> {
+  render = () => {
+    const { passenger } = this.props;
+    if (passenger == null) {
+      return null;
+    }
+    const title = idx(passenger, _ => _.title) || '';
+    const firstname = idx(passenger, _ => _.firstname) || '';
+    const lastname = idx(passenger, _ => _.lastname) || '';
+    const birthday = idx(passenger, _ => _.birthday) || '';
+    const nationality = idx(passenger, _ => _.nationality) || '';
+    const idNumber = idx(passenger, _ => _.travelDocument.idNumber) || '';
+    const insuranceType = startCase(
+      (idx(passenger, _ => _.insuranceType) || '').toLowerCase(),
+    );
+    return (
+      <SimpleCard style={StyleSheet.flatten(styles.card)}>
+        <Text style={styles.name}>
+          <Translation passThrough={`${title}. ${firstname} ${lastname}`} />
+        </Text>
+        <View style={[styles.row, styles.nationalityRow]}>
+          <RowItem
+            title={<Translation id="mmb.passenger.nationality.label" />}
+            value={<Translation passThrough={nationality.toUpperCase()} />}
+          />
+          <RowItem
+            title={<Translation id="mmb.passenger.birthday" />}
+            value={
+              <Translation passThrough={DateFormatter(birthday).format('L')} />
+            }
+          />
+        </View>
+        <SeparatorFullWidth />
+        <View style={[styles.row, styles.insuranceRow]}>
+          <RowItem
```

Comment:
I was a little bit confused by `RowItem` because it's actually a column. What about creating `Row` and `Column` components here and wrapping `SeparatorFullWidth` with vertical margin 15?

Reply:
It is an item in a row, the item uses column layout yes